src/services/strike_service.py

```diff
@@ -13,22 +16,126 @@ async def add_strike(
     user: discord.Member,
     severity: StrikeSeverity,
     reason: str,
+    author: discord.Member,
 ):
     # find user in DB
     db_user = users_database.get_user(user.id)
     if db_user is None:
         log_info_and_embed(
             logging_embed,
             logger,
-            f"User not found in database, creating new record",
+            "User not found in database, creating new record",
         )
-        db_user_id = users_database.add_user(user.id)
-        logger.info(f"Created user record in DB with id {db_user_id}")
-
-        db_user = create_empty_user(db_user_id, user.id, None, None)
+        db_user = users_database.add_user(user.id)
+        logger.info(f"Created user record in DB with id {db_user.user_id}")
```

Comment:
since we're logging this so many times throughout the code base, should we just throw this in `add_user`?